cellprofiler/modules/identifysecondaryobjects.py

```diff
@@ -114,6 +114,13 @@
 See the section "Measurements made by this module" below for the measurements
 that are produced by this module.
 
+*Note on saving images:* You can pass the modified objects along to the
```

Comment:
Additionally, see the paragraph beginning on line 88. Do you think it's redundant with the information here?